backuprestore.adoc

```diff
@@ -310,19 +310,16 @@ restore the LDAP server.
 
 * `zmschedulebackup` -- This command is used to schedule full backups,
 incremental backups, and deletion of old backups.
-* `zmbackup` -- This command executes full or incremental backup of the mail
+* `zmbackup` -- This command executes full or incremental backup of the {product-name} mailbox
 server. This is run on a live server, while the mailboxd process and the
 mailbox server are running. This command also has an option to manually
 delete old backups when they are no longer needed.
 * `zmbackupabort` -- This command stops a full backup that is in process.
 * `zmbackupabort -r` -- This command stops an ongoing restore.
 * `zmbackupquery` -- This command lists the information about ongoing and
 completed backups, including labels and dates.
-* `zmrestore` -- This command executes a full or incremental restore to the
-Zimbra mail server. The *zmrestore* command is performed on a server
-that is running.
-* `zmrestoreoffline` -- This command restores the Zimbra mail server when
-the mailboxd process is stopped.
+* `zmrestore` -- This command restores a backup to a running {product-name} mailbox server.
+* `zmrestoreoffline` -- This command restores a backup to a stopped {product-name} mailbox server.
```

Comment:
I prefer the original wording which makes it clearer that mailboxd should be stopped and not the whole server.

Reply:
@tmclane are you disagreeing here?

Reply:
No I am not disagreeing. Overlooked this reversion.

Reply:
Although I thought it had been updated. There seems to be another location at approximately line 804 with similar wording.